frontend/src/routes/(authenticated)/project/[project_code]/AddOrganization.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  import { BadgeButton } from '$lib/components/Badges';
+  import { FormModal } from '$lib/components/modals';
+  import { z } from 'zod';
+  import { _addProjectToOrg } from './+page';
+  import { Select } from '$lib/forms';
+
+  export let orgList;
+  export let projectId: string;
+
+  const schema = z.object({
+    orgId: z.string().trim()
+  });
+
+  type Schema = typeof schema;
+  let formModal: FormModal<Schema>;
+  $: form = formModal?.form();
+
+  async function openModal(): Promise<void> {
+    await formModal.open(async () => {
+      await _addProjectToOrg({
+        projectId,
+        orgId: $form.orgId,
+      })
+    });
+
+    // Error Handling
+  }
+
+</script>
+
+<BadgeButton variant="badge-success" icon="i-mdi-account-plus-outline" on:click={openModal}>
+  {'Add Organization'}
+</BadgeButton>
+
+<FormModal bind:this={formModal} {schema} let:errors>
+  <span slot="title">{'Choose Organization'}</span>
+  <Select
+    id="org"
+    label={'organization'}
+    bind:value={$form.orgId}
+    error={errors.orgId}
+    on:change
```

Comment:
Do you know what this does? I would like you to read https://svelte.dev/docs/element-directives#on-eventname and then leave a comment explaining what `on:change` is doing here, and why it's important (or, if you decide that it's not important and can be removed, why you believe it can be removed).

Reply:
`on:change` is a handler that listens to any changes made by user input, but in our case we are not passing any event handler to it and binding takes care of updating variables, meaning it can be removed.

Reply:
That's only partially correct. What do the docs say about the `on:` directive being used without a value?